### ISSUE TITLE
[EXE] Robustness of Machine Learning Models

### ISSUE THREAD

**Issue by hash-ir** [gimseng/99-ML-Learning-Projects#150]
### Learning Goals
- Create adversarial examples of input data using gradient descent (e.g Fast Gradient Attack)
- Train and visualize predictions of semantically same inputs (original and perturbed)
- Quantitative evaluation of the accuracy against training time (including adversary creation)

### Exercise Statement
Create adversarial examples and train a CNN on clean and perturbed data using PyTorch

### Prerequisites 
- Basics of creating and training neural networks in PyTorch
- Projected gradient descent, an idea about norm balls (L1, L2, L-inf)

### Data source/summary:
- Could be any image data (better explained and visualized with MNIST)

### (Optional) Suggest/Propose Solutions
I have the solution using PyTorch, will be happy to create a pull request to include the exercise statement/solution
